core/src/main/java/hudson/model/StringParameterDefinition.java

```diff
@@ -79,13 +78,11 @@ public String getDefaultValue() {
      * @return original or trimmed defaultValue (depending on trim)
      * @since TODO
      */
-    @Restricted(NoExternalUse.class)
     public String getDefaultValue4Build() {
```

Comment:
Not a great method name for actual API, why unrestrict this? What's the use case?